.github/workflows/ci.yml

```diff
@@ -43,12 +42,6 @@ jobs:
         mysql -h 127.0.0.1 -u root -proot -e 'CREATE DATABASE cakephp_test DEFAULT COLLATE=utf8mb4_general_ci;'
         mysql -h 127.0.0.1 -u root -proot -e 'CREATE DATABASE cakephp_comparisons;'
 
-    - name: Setup Postgres
-      if: matrix.db-type == 'pgsql'
-      run: |
-        psql -c 'CREATE DATABASE cakephp_test;' -U postgres
-        psql -c 'CREATE DATABASE cakephp_comparisons;' -U postgres
```

Comment:
should be able to use this to create another db.

Reply:
I tried a few different connection options and `createdb` as well without success. I'll try a `docker exec` approach next.

Reply:
I believe the port is mapped to 127.0.0.1 so make sure that's the host.

Reply:
That's what I would have expected too. But the builds from 8ca5f670191ba64e16d657b9e27a960e4530433b also failed because psql couldn't connect.